### DISCUSSION TITLE
Global Download Rate Throttle

### DISCUSSION THREAD

**Discussion by jult** [jesec/flood#546] [Ideas]
What is the unit? Monkeys? Bits, bytes? You really need to mention this at least somewhere. Nowhere could I find the syntax for these config options.
![Screenshot 2022-04-28 at 20-42-39 ↓ 13 7 kB_s ↑ 0 B_s - Flood](https://user-images.githubusercontent.com/7723770/165824172-8d360fee-53ce-44b8-a699-3d3f5741d65d.png)

**Comment by lnxd** [comment]
Apparently Bytes per second (B/s)

[client/src/javascript/components/sidebar/SpeedLimitDropdown.tsx](https://github.com/jesec/flood/blob/master/client/src/javascript/components/sidebar/SpeedLimitDropdown.tsx)

**Comment by trim21** [comment]
good catch
